src/sonic-config-engine/sonic-cfggen

```diff
@@ -138,6 +138,13 @@ def ip_network(value):
         return "Invalid ip address %s" % value
     return r_v.network
 
+def load_namespace_config(asic_name):
+    if not SonicDBConfig.isInit():
+        if is_multi_asic():
+            SonicDBConfig.load_sonic_global_db_config(namespace=asic_name)
```

Comment:
when we are parsing the host or device configuration on multi asic platform `asic_name` will be None.  In that case will `SonicDBConfig.load_sonic_global_db_config(namespace=None)` work ?